homeassistant/components/splunk/manifest.json

```
@@ -2,5 +2,11 @@
   "domain": "splunk",
   "name": "Splunk",
   "documentation": "https://www.home-assistant.io/integrations/splunk",
-  "codeowners": []
-}
+  "requirements": [
+    "splunk-data-sender==0.0.7",
+    "requests==2.24.0"
```

We don't need to depend on `requests`. It's a home assistant core requirement, and `splunk-data-sender` also depends on `requests`.

```suggestion
    "splunk-data-sender==0.0.7"
```